sphinx_doc/setup_guide.rst

```diff
@@ -1,328 +1,246 @@
+   # Set to start at boot time: 
+   sudo chkconfig memcached on 
 
 Database
 --------
 
-Using PostgreSQL as Database Backend:: 
+Install PostgreSQL::
 
-    # install PostgresSQL
+    # Add line to yum repository: 
+    echo 'exclude=postgresql*' | sudo tee -a /etc/yum.repos.d/CentOS-Base.repo
 
-    postgres=# create user django; 
-    postgres=# create database django;
-    postgres=# grant all on database django to django;
-    # connect to database django
-    postgres=# \c django 
-    # create extension hstore
-    postgres=# create extension hstore;
-
-    # config in pg_hba.conf
-    pip install psycopg2==2.6
-
-* Install PostgreSQL ( `postgresql reference`_ )
-
-  .. _postgresql reference: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-centos-7
-* Create the user and database for this application
-* Allowing connection from the database user by setting your ``/var/lib/pgsql/9.x/data/pg_hba.conf``
-* Install pip package psycopg2. Path of pg_config binary may need to be exported. 
+    # Install the PostgreSQL Global Development Group (PGDG) RPM file:
+    sudo yum -y localinstall http://yum.postgresql.org/9.5/redhat/rhel-6-x86_64/pgdg-centos95-9.5-2.noarch.rpm
+    
+    # Install PostgreSQL 9.5:
+    sudo yum -y install postgresql95-server postgresql95-contrib postgresql95-devel
+    
+    # Initialize (uses default data directory: /var/lib/pgsql):
+    sudo service postgresql-9.5 initdb   
+    
+    # Startup at boot:
+    sudo chkconfig postgresql-9.5 on
+    
+    # Control:
+    # sudo service postgresql-9.5 <command>
+    # 
+    # where <command> can be:
+    #  
+    #     start   : start the database.
+    #     stop    : stop the database.
+    #     restart : stop/start the database; used to read changes to core configuration files.
+    #     reload  : reload pg_hba.conf file while keeping database running. 
+    
+    # Start:
+    sudo service postgresql-9.5 start
 
-::
+    #
+    #  (To remove everything: sudo yum erase postgresql95*)
+    #
+    
+    # Create django database and user:
+    sudo su - postgres
+    psql
+    
+    # At the prompt 'postgres=#' enter:
+    create database django;
+    create user django;
+    grant all on database django to django;
+    
+    # Connect to django database:
+    \c django
+    
+    # Create extension hstore:
+    create extension hstore;
 
+    # Config in pg_hba.conf:
+    cd <virt-env> 
+    export PATH=/usr/pgsql-9.5/bin:$PATH
     pip install psycopg2==2.6
-    export PATH=/usr/pgsql-9.x/bin:"$PATH"
 
-Migration db schema to PostgreSQL 
+ 
+Migrate Schema to to PostgreSQL
+------------------------------- 
 
-::
+Run migrate::
 
+    cd <virt-env>
```

Comment:
At the the section 'Python 2.7.8' we activate the virtualenv.  From then 
on we use its bin, lib, and include, instead of the global ones.   
<virt-env> is the parent directory to the virtualenv bin, lib, and 
include dirs and where we copy the django-blast project to, from the 
local git repository.

So when we run migrate, we cd into <virt-env> which is where manage.py 
is, and the virtualenv is still
active.

Did I answer you question, and is this part all right?

Thanks

Joseph



On 06/29/2016 11:44 AM, childers wrote:
>
> In sphinx_doc/setup_guide.rst 
> <https://github.com/NAL-i5K/django-blast/pull/2#discussion_r68972243>:
>
> >
> > +    cd <virt-env>
>
> Basic question, is this essentially the same as running
> source /py-2.7/bin/activate
> then running manage.py migrate?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/NAL-i5K/django-blast/pull/2/files/6fb9f04451a3b598752457f2347e5bc6554e40e0#r68972243>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ATKh8_AI6cUhwIH9TmRGINxD4BM7IFV4ks5qQpLrgaJpZM4JAUXb>.
>